source/whats-new.txt

```diff
@@ -36,6 +37,18 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.5:
+
+What's New in 5.5
+-----------------
+
+New features of the 5.5 {+driver-short+} release include:
+
+- The driver now accurately detects Function-as-a-Service (FaaS)
+  environments in AWS.
+- You must upgrade ``mongodb-client-encryption`` to version 2.8.0 or
+  later if you want to create a QE collection while using this driver version.
```

Comment:
Issue:

1) As recommended by [the style guide](https://www.mongodb.com/docs/meta/style-guide/style/abbreviations/), explain the terms before using abbreviations.

2 I think) QE collection is not the official name for encrypted collection; instead, use "encrypted collection" as referred to on this page.

Suggestion:

1) Use "Queryable Encryption".

2) Use "encrypted collection" as it is referred to on [this page](https://www.mongodb.com/docs/manual/core/queryable-encryption/fundamentals/manage-collections/).

Both suggestions incorporated:
"You must use ... v2.8.0 or later to encrypt collections by using the Queryable Encryption feature."

Reply:
I guess the "in this this driver version" is implicit

Reply:
> I guess the "in this this driver version" is implicit

I agree, and thanks for catching the mistake in my suggestion.

I think there is also nuance here in which omitting the driver version is important. From what I know so far, the previous version of the driver won't be able to create encrypted collections when using older versions of mongodb-client-encryption either as that functionality will be removed from the server too.